rust-runtime/aws-smithy-http/src/body.rs

```diff
@@ -250,6 +250,22 @@ impl From<hyper::Body> for SdkBody {
     }
 }
 
+impl From<lambda_http::Body> for SdkBody {
+    fn from(body: lambda_http::Body) -> Self {
+        let hyper_body = match body {
+            lambda_http::Body::Empty => hyper::Body::empty(),
+            lambda_http::Body::Text(s) => hyper::Body::from(s),
+            lambda_http::Body::Binary(v) => hyper::Body::from(v),
+        };
+
+        SdkBody {
+            inner: Inner::Streaming { inner: hyper_body },
+            rebuild: None,
+            callbacks: Vec::new(),
+        }
+    }
+}
+
```

Comment:
we definitely don't want an un-gated dependency on `lambda_http` in `aws-smithy-http`. Can we keep these converters isolated? I'd probably suggest refactoring the way this works so that there is a "labmda" module that can be conditionally enabled, preferably in the server crate

Reply:
I thought this might be the case, I added the code to get the feedback, I'll see about doing a lambda module.